### ISSUE TITLE
[question] CLICOLOR_FORCE (still) not working in Conan 2.0

### ISSUE THREAD

**Issue by santhonisz** [conan-io/conan#13181]
### What is your question?

Hi there,

I have been evaluating Conan 2 (beta 10) in our CI environment (Buildkite) and have noticed that the fix for #11892 doesn't appear to be working i.e. there is still no colored output when defining the `CLICOLOR_FORCE` environnment variable under Conan 2.x. 

I've done a fair bit of debugging and think I am across the cause of the problem. The call to the Colorama `init` function is different compared to 1.x when attempting to force color:

**Conan 1.x**
https://github.com/conan-io/conan/blob/aef2eea04fa5c457ae28950b836264d969ec1322/conans/cli/output.py#L149-L151

**Conan 2.x**
https://github.com/conan-io/conan/blob/da56e966a92243f350e3715055a8c6f13dfad822/conans/client/userio.py#L41-L43

With that in mind I performed testing in CI independent of Conan using the following script. When using no args for the `init` function as in Conan 2.x there was no color output in CI, where if I switched it to be the same as in Conan 1.x the colored output returns:
```python
import colorama
from colorama import Fore

# no colored output
colorama.init()
print(Fore.RED + 'some red text')

deinit()

# produces colored output
colorama.init(convert=False, strip=False)
print(Fore.RED + 'some red text')
```

![image](https://user-images.githubusercontent.com/29723862/220259199-46f5be23-e21b-4afa-a6a8-876dbd942a84.png)

While that suggests the apparent fix is updating the call to the `init` function in Conan 2.x, taking a look at the [README ](https://github.com/tartley/colorama/blob/master/README.rst) for Colorama revealed that there's a newer approach than using the `init` function available in later versions :

> Most users should depend on colorama >= 0.4.6, and use just_fix_windows_console.

Indeed, the following does appear to work as advertised:

```python
import colorama
from colorama import Fore

# produces colored output
colorama.just_fix_windows_console()
print(Fore.RED + 'some red text')
```

So could/should the fix to Conan 2 be to bump the dependency to `colorama>=0.4.6, <0.5.0` and replace the call to `colorama.init()` with `colorama.just_fix_windows_console()`? 

### Have you read the CONTRIBUTING guide?

- [X] I've read the CONTRIBUTING guide

**Comment by memsharded** [comment]
Closed by https://github.com/conan-io/conan/pull/13183

**Comment by memsharded** [comment]
Thanks for reporting, and thanks for the detailed investigation. We have fixed in for next 2.0 launch.

Regarding the suggestion to use just the latest release 0.4.6, it will probably be a bit too restrictive, and will cause conflicts to users that rely on previous versions of colorama, so maybe better to keep the range a bit wider.
